train.py

```diff
@@ -27,7 +27,7 @@
 WAVENET_PARAMS = './wavenet_params.json'
 STARTED_DATESTRING = "{0:%Y-%m-%dT%H-%M-%S}".format(datetime.now())
 SAMPLE_SIZE = 100000
-
+L2_REGULARIZATION_STRENGTH=0.
```

Comment:
Then, it might make sense to have a second JSON file with training parameters: L2 regularization, learning rate, learning rate decay, dropout rate, ...